packages/replay-internal/src/util/getPrivacyOptions.ts

```diff
@@ -25,7 +25,7 @@ function getOption(selectors: string[], defaultSelectors: string[]): string {
  * Returns privacy related configuration for use in rrweb
  */
 export function getPrivacyOptions({ mask, unmask, block, unblock, ignore }: GetPrivacyOptions): GetPrivacyReturn {
-  const defaultBlockedElements = ['base[href="/"]'];
+  const defaultBlockedElements = ['base[href="/"]', 'iframe[srcdoc]:not([src])'];
```

Comment:
sidenote, we should probably block all `base` not just with `href` attribute

Reply:
@billyvg added it